src/main/resources/assets/js/components/NewAppModalComponent.jsx

```diff
@@ -132,9 +150,14 @@ var NewAppModalComponent = React.createClass({
     if ("instances" in modelAttrs) {
       modelAttrs.instances = parseInt(modelAttrs.instances, 10);
     }
-
+    if ("privileges" in modelAttrs) {
+      if (modelAttrs.privileges === "true") {
+        modelAttrs.instances = true;
```

Comment:
Shouldn't be modelAttrs.instances here?

You could do this without if statement:
```
modelAttrs.privileges = modelAttrs.privileges === "true"
```